### ISSUE TITLE
I cannot set vehicle properties at runtime

### ISSUE THREAD

**Issue by RuanLucasGD** [FlaxEngine/FlaxEngine#1348]
**Issue description:**
Set any value in runtime don't work, example code:

```
var vehicleEngine = Engine;
vehicleEngine.MaxRotationSpeed = maxSpeed;
vehicleEngine.MaxTorque = Acceleration;
vehicleEngine.MOI = Moi;
Vehicle.Engine = vehicleEngine;
```` 

**Flax version:**
1.6

**Comment by mafiesto4** [comment]
You can call `Setup()` to update the vehicle (alll properties) but it's causing the full physics state recreation so not a smooth change. I'll add soft values update for easier dynamic configuration.

**Comment by mafiesto4** [comment]
Fixed in 36dca16991ddcf3504834402d495c53a1d85dfc1.
